sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -662,6 +662,23 @@ def _add_argparse_args(cls, parser):
                          'in the pipeline when running using the Fn API.'))
 
 
+class FlinkOptions(PipelineOptions):
```

Comment:
I wonder if this should go somewhere under runners.

Reply:
Was wondering about this too but I saw that we store all the PipelineOptions here, e.g. GCE, DirectRunner.

Reply:
Yeah, I guess it doesn't make it any worse. (The way we do things here has always felt slightly unpythonic to me, but cleaning this up is low on the priority list...)

Reply:
I'm open to cleaning it up. Could move this into a FlinkOptions file in the `runners/flink` package, which doesn't exist yet.